packages/now-cli/src/util/client.ts

```diff
@@ -80,20 +72,21 @@ export default class Client extends EventEmitter {
       Object.assign(opts, {
         body: JSON.stringify(opts.body),
         headers: Object.assign({}, opts.headers, {
-          'Content-Type': 'application/json'
-        })
+          'Content-Type': 'application/json',
+        }),
       });
     }
 
     opts.headers = opts.headers || {};
     opts.headers.authorization = `Bearer ${this._token}`;
     opts.headers['user-agent'] = ua;
 
+    const url = `${apiUrl ? '' : this._apiUrl}${_url}`;
     return this._output.time(
-      `${opts.method || 'GET'} ${apiUrl ? '' : this._apiUrl}${_url} ${JSON.stringify(
-        opts.body
-      ) || ''}`,
-      this._agent.fetch(_url, opts as AgentFetchOptions)
+      `${opts.method || 'GET'} ${
+        apiUrl ? '' : this._apiUrl
+      }${_url} ${JSON.stringify(opts.body) || ''}`,
```

Comment:
Seems like this time call could be simplified since you already created `url` above:

```js
this._output.time(`${opts.method || 'GET'} ${url} ${JSON.stringify(opts.body) || ''}`)
```